qbo_api.gemspec

```diff
@@ -25,8 +25,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'dotenv'
   spec.add_development_dependency 'vcr'
   spec.add_development_dependency 'amazing_print'
-  spec.add_runtime_dependency 'faraday', '~> 1.10.0'
-  spec.add_runtime_dependency 'faraday_middleware'
+  spec.add_runtime_dependency 'faraday', '>= 1.10.0'
   spec.add_runtime_dependency 'faraday-detailed_logger'
```

Comment:
Have you tested this branch tests pass with both Faraday 1.x and 2.x? And that you can bundle it in your apl with 2.x?

Reply:
I just ran the tests. Wish there was some sort of CI matrix set up. 

I wasn't really considering backwards compatible changes. I was just trying to get your PR over the finish line so we can upgrade to Faraday 2 😅

I'll check Faraday >= 1.10 & < 2.0 today

Reply:
I did just run green tests under Faraday 1.10.2. 

*re: CI matrix:* Soon, but I don't have the bandwidth currently.

I'd release as a version `3.00` so one could always drop back to v. `~> 2.00`.

Reply:
Thank you. I'll update the version.